src/components/screens/bookmarks/addBookmark/addBookmark.js

```diff
@@ -105,11 +105,11 @@ const AddBookmark = ({
   }, []);
 
   useEffect(() => {
-    if (account.data.summary) {
-      const username = account.data.dpos?.delegate?.username ?? '';
+    if (account.data?.summary) {
+      const username = account.data?.dpos?.delegate?.username ?? '';
```

Comment:
This is not needed, if you null checked the live above, you don't need the optional chaining here.